src/Abstractions/NexusMods.Abstractions.Games.Diagnostics/Writer/DiagnosticWriterMode.cs

```diff
@@ -0,0 +1,20 @@
+using JetBrains.Annotations;
+
+namespace NexusMods.Abstractions.Diagnostics;
+
+/// <summary>
+/// Represents the valid modes of a <see cref="IDiagnosticWriter"/>.
+/// </summary>
+[PublicAPI]
+public enum DiagnosticWriterMode
```

Comment:
Should be marked as `byte`.

Reply:
I used to put `byte` on every enum out of habit writing binary serialization stuff, but in most cases it doesn't matter due to alignment.